### PR TITLE
fix(arch-wiki): fix a bunch of broken stuff

### PR DESCRIPTION
**Pull request by somerand0mcat**
i was testing out the arch wiki and i found a bunch of broken stuff
image
image
which i fixed
image
image

### CODE REVIEW

File: styles/arch-wiki/catppuccin.user.css [hunk 1]
```diff
@@ -429,6 +425,42 @@
...
+      background-color: @base !important;
+      color: @text !important;
+      border-color: @base !important;
+    }
+      
+    .mw-body {
+      background-color: @mantle !important;
+      color: @text !important;
+      border-color: @base !important;
+      border-radius: 10px;
```
Comment (isabelroses) [inline styles/arch-wiki/catppuccin.user.css:439]: Let's avoid opinionated changes like this
```suggestion
```

File: styles/arch-wiki/catppuccin.user.css [hunk 2]
```diff
@@ -429,6 +425,42 @@
...
+      background-color: @mantle !important;
+    }
+      
+    .vector-sticky-pinned-container::after {
+      background: @base !important;
+    }
+      
+    .cdx-search-input {
+      border-color: @mantle !important;
+      border-width: 3px
```
Comment (isabelroses) [inline styles/arch-wiki/catppuccin.user.css:458]: same as prior comment
```suggestion
```

File: styles/arch-wiki/catppuccin.user.css [hunk 3]
```diff
@@ -429,6 +425,40 @@
...
     }
+      
+    .cdx-button {
+      background-color: @base !important;
+      color: @text !important;
+      border-color: @base !important;
+    }
+      
+    .mw-body {
+      background-color: @mantle !important;
```
Comment (isabelroses) [inline styles/arch-wiki/catppuccin.user.css:436]: Content that is in the middle should be base following the style guide, perhaps make the header crust and the outer background mantle?
```suggestion
      background-color: @base !important;
```

Comment (somerand0mcat) [reply styles/arch-wiki/catppuccin.user.css:436]: i think thats a good idea yeah

**Review by isabelroses — COMMENTED** [review]
This is going to need some changes
image

**Comment by somerand0mcat** [comment]
@isabelroses fixed
image

**Review by isabelroses — APPROVED** [review]

**Pull request merged**